src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -304,9 +304,7 @@ class KotlinPoetTest {
         |
         |import java.lang.String
         |
-        |fun String.shrink(): String {
-        |  return substring(0, length - 1)
-        |}
+        |fun String.shrink(): String = substring(0, length - 1)
```

Comment:
What is the point of inferring types in generated code? It doesn't really add any value. Moreover,  I think that it is good in general for generated code to be as explicit as possible.